src/commands/diff.ts

```diff
@@ -60,8 +62,7 @@ export default class Diff extends Command {
   */
   async run(): Promise<void> {
     const { args, flags } = this.parse(Diff);
-    /* eslint-disable-next-line @typescript-eslint/no-non-null-assertion */
-    const [documentation, hub, token] = [flags.doc!, flags.hub, flags.token!];
+    const [documentation, hub, token] = [flags.doc, flags.hub, flags.token];
```

Comment:
Question similar here to `?`, 
change from `flags.doc!` to `flags.doc` is because param `doc` is not required anymore, right ?

Reply:
Yes, due to [this change](https://github.com/bump-sh/cli/pull/221/files#diff-36ff7c9416eb5f08ba70a0105cda4b2aec01654ee6af9fed81d2363805fd43daR48-R50), the `--doc` and `--token` flags are not required anymore, so we can't be sure that the values are defined or not.

The previous usage of `!` was kind of a “hack” because oclif types are not well defined for **required** flags, and thus we needed to “tell the typescript compiler” that those attributes are always defined.

With the changes now, the attributes can be undefined thus we don't need the `!` anymore